apps/home/src/components/AppTile/styles/AppTile.module.scss

```
@@ -0,0 +1,39 @@
+  }
+
+  &:focus,
+  &:focus-visible {
+    outline: 2px solid $focus;
+    outline-offset: -2px;
+  }
+}
+
+.label {
+  @include type-style("body-compact-01");
+  margin: 0;
```

Stylelint (17.11.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)